pom.xml

```diff
@@ -75,6 +88,11 @@
             <artifactId>hamcrest-core</artifactId>
             <version>${hamcrest.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
Switched from a used, but previously undeclared dependency on commons-lang to a declared dependency on commons-lang3.